<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uploadDiary</class>
 <widget class="QWidget" name="uploadDiary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="titleLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入日记标题</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="destinationLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入目的地</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发布日记</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="contentTextEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入日记内容</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
